### PR TITLE
rating-component

### PR DESCRIPTION
**Pull request by anshtripathi01**
**Please review it**
rating

### CODE REVIEW

File: Components/Rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+.rating-container{
+    display: flex;
+justify-content: center;
+align-items: center;
+flex-direction: column;
+    width: 20rem;
+    height: 12rem;
+    box-shadow: 0px 0px 3px 1px rgba(0,0,0,0.1);
+}
```
Comment (ravikumar0403) [inline Components/Rating/rating.css:11]: Make sure the code formatting is correct. You can use extensions for formatting your code.

**Review by ravikumar0403 — APPROVED** [review]
Looks good. Left a couple of sanity comments

File: Components/Rating/rating.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (bharati-21) [inline Components/Rating/rating.html:7]: Suggestion - can add author and description meta tags.

**Review by bharati-21 — APPROVED** [review]
Looks good to be merged!

**Comment by anshtripathi01** [comment]
Thanks!!! I'll add.

**Pull request merged**